kanatrans/src/adapter/processor/lex_lookup.rs

```diff
@@ -26,8 +26,11 @@ where
 
 impl Executor for LexLookupExecutor {
     fn execute(&self, word: &str) -> Result<Vec<String>> {
-        let lexs = LEXICON.lookup(&word.to_lowercase(), None).map_err(|e| anyhow!(e))?;
-        Ok(lexs)
+        match LEXICON.lookup(&word.to_lowercase(), None) {
+            Ok(lexs) if lexs.is_empty() => Err(anyhow!("cannot convert to ARPAbet")),
```

Comment:
Import `anyhow::bail` and use `bail!` instead of `Err(anyhow!())`.

```suggestion
            Ok(lexs) if lexs.is_empty() => bail!("cannot convert to ARPAbet"),
```

Reply:
Also fixed two line below that line.